07_errors/n0npax/errors.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+)
+
+// Error codes
+const (
+	ErrInvalidInputCode  = 400
```

Comment:
I'd suggest naming these code by keeping `Err` and `Code` together, either `InvalidInputErrCode` or `ErrCodeInvalidInput`. That allows you to grep for `ErrCode` to find all the error codes. I think I prefer the first one as the actual error name appears first which is what you most want to see, but up to you.

Reply:
`sed -i 's/ErrInvalidInputCode/ErrCodeInvalidInput/g; s/ErrNotFoundCode/ErrCodeNotFound/g; s/ErrInternalErrorCode/ErrCodeInternal/g' *.go
`